content/en/blog/_posts/2022-11-21-devicemanager-ga.md/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: Graduation of Device Manager to GA!'
+date: 2021-11-21
```

Comment:
We should change this to the assigned date once we know it.

Reply:
Sure, I will mark it as a TODO for now till we have more information. I assumed that this would be the date this was written and put that date here.